heartbeat/podman-etcd

```diff
@@ -1194,29 +1208,31 @@ podman_start()
 				case "$start_resources_count" in
 				1)
 					ocf_log debug "peer not starting: ensure we can start a new cluster"
-					if [ "$revision_compare_result" != "older" ]; then
+					if [ "$revision_compare_result" != "older" ] || [ "$revision_compare_result" != "only_peer" ]; then
 						# If our revision is the same as or newer than the peer's last saved
 						# revision, and the peer agent isn't currently starting, we can
 						# restore e-quorum by forcing a new cluster.
 						set_force_new_cluster
 					else
-						ocf_log err "local revision is older and peer is not starting: cannot start"
-						ocf_exit_reason "local revision is older and peer is not starting: cannot start"
+						ocf_log err "local revision is older or empty and peer is not starting: cannot start"
+						ocf_exit_reason "local revision is older or emptyand peer is not starting: cannot start"
 						return "$OCF_ERR_GENERIC"
 					fi
 					;;
 				2)
 					ocf_log info "peer starting"
 					if [ "$revision_compare_result" = "newer" ]; then
 						set_force_new_cluster
-					elif [ "$revision_compare_result" = "older" ]; then
+					elif [ "$revision_compare_result" = "older" ] || [ "$revision_compare_result" = "only_peer" ]; then
 						ocf_log info "$NODENAME shall join as learner"
 						JOIN_AS_LEARNER=true
 					else
 						if [ "$(attribute_node_cluster_id get)" = "$(attribute_node_cluster_id_peer)" ]; then
 							ocf_log info "same cluster_id and revision: start normal"
+						else if [ "$revision_compare_result" = "only_local" ] && [ "$(attribute_node_cluster_id get)" != ""] && [ "$(attribute_node_cluster_id get)" != "null" ]; then
```

Comment:
A couple of minor notes on this
* you can use elif here
* there's a missing space before a closing bracket "]"

Moreover, this seems wrong to me. We have 2 starting resources with different cluster-ids, they cannot start normally.